app/javascript/controllers/dynamic_form_controller.js

```diff
@@ -1,7 +1,17 @@
 import { Controller } from "@hotwired/stimulus"
 
 export default class extends Controller {
-  static targets = ["link"]
+  static targets = ["link", "help"]
+
+  static values = {
+    help: Object,
+  }
+
+  updateHelp(event) {
```

Comment:
This feels like it belongs in a different controller.  The small/single purpose of this one is to watch for a field to change and click an anchor.  Simulating a non-js flow where the user changes a select or something, then clicks "update form".

Reply:
Ok. I can move it to its own controller. I interpreted it as a dynamic form feature, which is why I put it here.